src/main/java/com/es/phoneshop/model/cart/DefaultCartService.java

```diff
@@ -0,0 +1,53 @@
+package com.es.phoneshop.model.cart;
```

Comment:
Please don't forget to format the code:

Ctrl + Alt + L in IntelliJ IDEA to format the code in the file (or configure auto-formatting on file save).

Please press Ctrl + Alt + L and check how the formatting changes - this will give you some insights on missing spaces, extra spaces, etc.